### PR TITLE
assets folder is the source of truth

### PR DESCRIPTION
**Pull request by mag-da-lenka**
# Description

Please provide a short summary explaining what this PR is about.

Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

Checklist not needed. 
The naming of the new assets will obviously break some code but that's what you all know about. 

- [x] I have performed a self-review of my own code
- [x] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [x] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [x] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by rebeccahockley — APPROVED** [review]
Reviewed and approved 👍🏼

**Comment by mag-da-lenka** [comment]
Everything up and running. Checks done. All content updated and formatted.

File: packages/client/src/components/Main/Main.css [hunk 1]
```diff
@@ -5,7 +5,7 @@ main {
       rgba(28, 65, 64, 0.8) 0.01%,
       rgba(7, 27, 32, 0.5) 100%
     ),
-    url('../../../public/assets/images/body-background-image-concrete.png');
+    url('../../../public/assets/images/concrete-bw.png');
```
Comment (miltonhultgren) [inline packages/client/src/components/Main/Main.css:8]:
```suggestion
    url('/assets/images/concrete-bw.png');
```

Once the app is published the URLs for assets need to be relative to the web sever.

**Review by miltonhultgren — APPROVED** [review]
I left one little thing to fix but other than that it looks good to me!

File: packages/client/src/components/Main/Main.css (hunk 1 above)
Comment (mag-da-lenka) [reply packages/client/src/components/Main/Main.css]: @miltonhultgren Sorry I was so fast-forward I pressed this damn button. Could we undo it here? 😵‍💫

Comment (mag-da-lenka) [reply packages/client/src/components/Main/Main.css]: Could you make an opposite suggestion so that I can click it and merge the thing?

Comment (mag-da-lenka) [inline packages/client/src/components/Main/Main.css]: url('../../../public/assets/images/concrete-bw.png');
	    url('/assets/images/concrete-bw.png');

Comment (mag-da-lenka) [inline packages/client/src/components/Main/Main.css]: url('../../../public/assets/images/concrete-bw.png');
    url('/assets/images/concrete-bw.png');

**Pull request merged**